.changeset/honest-weeks-shake.md

```diff
@@ -0,0 +1,58 @@
+---
+'@primer/react-brand': minor
+---
+
+New ComparisonTable component available
+
+```
+<ComparisonTable featuredColumn={1} heading="GitHub vs Jenkins">
+    <ComparisonTable.Row>
+      <ComparisonTable.Cell>Use case</ComparisonTable.Cell>
+      <ComparisonTable.Cell>GitHub</ComparisonTable.Cell>
+      <ComparisonTable.Cell>Jenkins</ComparisonTable.Cell>
+    </ComparisonTable.Row>
+    <ComparisonTable.Row>
+      <ComparisonTable.Cell>Automation & CI/CD</ComparisonTable.Cell>
+      <ComparisonTable.Cell>
+        <>
```

Comment:
Where did we end up on the conversation around using ReactElement | ReactElement[] to avoid the need for fragments here?

Reply:
These are old examples. Removed in a [recent commit](https://github.com/primer/brand/pull/108/commits/8540610fefd431f4334e3b0ade70e241c3f1fc29). Using `PropsWithChildren` seems to work okay and compile correctly.